### PR TITLE
Akonadi and plasmashell updates

### PR DESCRIPTION
**Pull request by curiosityseeker**

### CODE REVIEW

File: apparmor.d/groups/akonadi/akonadi_akonotes_resource [hunk 1]
```diff
@@ -38,10 +38,12 @@ profile akonadi_akonotes_resource @{exec_path} {
   owner @{user_config_dirs}/kdedefaults/kwinrc r,
   owner @{user_config_dirs}/kdeglobals r,
   owner @{user_config_dirs}/kwinrc r,
+  
+  @{sys}/devices/pci[0-9]*/**/{device,subsystem_device,subsystem_vendor,uevent,vendor} r,
```
Comment (roddhjav) [inline apparmor.d/groups/akonadi/akonadi_akonotes_resource:42]: This is already present in `abstractions/dri-enumerate`

File: apparmor.d/groups/akonadi/akonadi_mailfilter_agent [hunk 2]
```diff
@@ -16,6 +16,7 @@ profile akonadi_mailfilter_agent @{exec_path} {
   include <abstractions/mesa>
   include <abstractions/nameservice-strict>
   include <abstractions/qt5>
+  include <abstractions/user-tmp>
```
Comment (roddhjav) [inline apparmor.d/groups/akonadi/akonadi_mailfilter_agent:19]: Could you explain what tmp file is needed. It is usually not needed to give full access to user-tmp.

File: apparmor.d/groups/akonadi/akonadi_migration_agent [hunk 3]
```diff
@@ -22,6 +22,7 @@ profile akonadi_migration_agent @{exec_path} {
   @{exec_path} mr,
 
   /usr/share/hwdata/*.ids r,
+  /usr/share/icons/{,**} r,
```
Comment (roddhjav) [inline apparmor.d/groups/akonadi/akonadi_migration_agent:25]: Already present in the `freedesktop.org` abs.

File: apparmor.d/groups/kde/plasmashell [hunk 4]
```diff
@@ -49,11 +50,18 @@ profile plasmashell @{exec_path} {
   /usr/share/krunner/{,**} r,
   /usr/share/konsole/ r,
   /usr/share/akonadi/firstrun/{,*} r,
+  /usr/share/pipewire/client.conf r,
```
Comment (roddhjav) [inline apparmor.d/groups/kde/plasmashell:53]: Just give the `audio` abs

File: apparmor.d/groups/kde/plasmashell [hunk 5]
```diff
@@ -142,9 +155,13 @@ profile plasmashell @{exec_path} {
...
   owner @{PROC}/@{pid}/mounts r,
+  owner @{PROC}/@{pid}/{cgroup,cmdline,stat,statm} r,
+  owner @{PROC}/@{pid}/attr/current r,
 
-  /dev/shm/ r,
-  /dev/tty r,
+        /dev/shm/ r,
+        /dev/tty r,
+        /dev/ptmx rw,
+  owner /dev/pts/[0-9]* rw,
```
Comment (roddhjav) [inline apparmor.d/groups/kde/plasmashell:164]: Use the consoles abs instead.

File: apparmor.d/groups/akonadi/akonadi_akonotes_resource (hunk 1 above)
Comment (curiosityseeker) [reply apparmor.d/groups/akonadi/akonadi_akonotes_resource:42]: Oops. I had collected these rules over several weeks and missed that you added dri-enumerate in the meantime.

File: apparmor.d/groups/akonadi/akonadi_mailfilter_agent (hunk 2 above)
Comment (curiosityseeker) [reply apparmor.d/groups/akonadi/akonadi_mailfilter_agent:19]: I'll check that.

**Comment by roddhjav** [comment]
Thanks a lot. Merged!

**Pull request merged**